omnibus-arbitrum-mainnet.toml

```diff
@@ -1,9 +1,10 @@
 name = "synthetix-omnibus"
-version = "37"
+version = "38"
 description = "Includes the full synthetix system with configurations applied"
 preset = "main"
```

Comment:
please add `deployers`

Reply:
yeah this was added in a different branch, doing it now

Reply:
https://github.com/Synthetixio/synthetix-deployments/pull/534